src/ensembl/src/content/app/browser/browser-bar/BrowserBar.tsx

```diff
@@ -119,6 +126,14 @@ export const BrowserBar: FunctionComponent<BrowserBarProps> = (
     props.toggleBrowserNav();
   };
 
+  const shouldShowBrowserTabs = () => {
+    return (
+      props.activeGenomeId &&
+      (props.isTrackPanelOpened ||
+        props.breakpointWidth === BreakpointWidth.LARGE)
+    );
+  };
```

Comment:
Now that I noticed how `shouldShowBrowserTabs` is used, I think it should just be changed to a boolean constant instead of the function:

```
const shouldShowBrowserTabs = (
      props.activeGenomeId &&
      (props.isTrackPanelOpened ||
        props.breakpointWidth === BreakpointWidth.LARGE)
 );
```

Same goes for shouldShowBrowserInfo, I think.

Reply:
Done for both